packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -95,6 +95,9 @@ class GestureRecognizerFactoryWithHandlers<T extends GestureRecognizer> extends
 /// If this widget has a child, it defers to that child for its sizing behavior.
 /// If it does not have a child, it grows to fit the parent instead.
 ///
+/// By default a GestureDetector with an invisible child ignores touches,
+/// this behavior can be controlled with [behavior].
```

Comment:
semicolon or period, not colon, since the second part is a distinct sentence with its own subject.